flaskr.py

```diff
@@ -1,16 +1,31 @@
-from flask import Flask
+from flask import Flask, render_template, url_for
```

Comment:
Flask --> A class 
render_template('<template.html>',  <variable_name>=<value>) --> To render template from **templates**
url_for('static', filename='style.css') --> To provide exact location of route